src/PrestaShopBundle/Form/Admin/Sell/Product/Pricing/SpecificPriceType.php

```diff
@@ -208,11 +208,12 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
                 'default_empty_data' => 0.0,
             ])
             ->add('leave_initial_price', CheckboxType::class, [
-                'label' => $this->trans('Leave initial price', 'Admin.Catalog.Feature'),
+                'label' => $this->trans('Apply a discount to the initial price', 'Admin.Catalog.Feature'),
+                'help' => 'For customers meeting the conditions, the initial price will be crossed out and the discount will be highlighted',
```

Comment:
There's a period missing at the end of this sentence, " highlighted**.**" 

(I couldn't make a commit suggestion for some reason...)